src/components/Onboarding/Screens/Apps/AgreementSettings/index.js

```diff
@@ -181,10 +181,10 @@ function AgreementSettings() {
           label={
             <Fragment>
               Settlement Period
-              <Help hint="What is Settlement Period?">
-                <strong>Settlement Period</strong> is the amount of time the
-                proposal's creator has to either accept a settlement or raise
-                the dispute to{' '}
+              <Help hint="What is the Settlement Period?">
+                The <strong>Settlement Period</strong> is the amount of time the
+                proposal author has to either accept a settlement or raise the
+                dispute to{' '}
```

Comment:
Realized that we could make use of the `HelpTip` for all the garden settings as well. 

Will create an issue for it